tests/context.py

```diff
@@ -19,6 +22,46 @@ def _Pow(a, b):
     return Pow(a, b, evaluate=False)
 
 
+def get_simple_examples(func):
```

Comment:
ooooh thanks for breaking these tests apart. :kermit:

Reply:
they were very much duplicated in multiple places 😂

Reply:
they were out of control. tests are all passing when I run locally. 👍